src/webgpu/shader/execution/shader_io/fragment_builtins.spec.ts

```diff
@@ -403,6 +450,13 @@ function computeFragmentSampleIndex({ sampleIndex }: FragData) {
   return [sampleIndex, 0, 0, 0];
 }
 
+/**
+ * Computes 'builtin(front_facing)'
+ */
+function computeFragmentFrontFacing({ frontFacing }: FragData) {
+  return [frontFacing ? 0 : 1, 0, 0, 0];
```

Comment:
corresponding `outputCode: 'vec4f(select(0.0, 1.0, fin.frontFacing), 0, 0, 0)'`

```suggestion
  return [frontFacing ? 1 : 0, 0, 0, 0];
```

Reply:
done